dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/CommentOnWorkflowActionlet.java

```diff
@@ -1,58 +1,114 @@
 package com.dotmarketing.portlets.workflows.actionlet;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
 import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.HibernateUtil;
 import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotHibernateException;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
 import com.dotmarketing.portlets.workflows.model.WorkflowActionClassParameter;
 import com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException;
 import com.dotmarketing.portlets.workflows.model.WorkflowActionletParameter;
 import com.dotmarketing.portlets.workflows.model.WorkflowComment;
 import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
 import com.dotmarketing.portlets.workflows.model.WorkflowStep;
+import com.dotmarketing.portlets.workflows.model.WorkflowTask;
 import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.VelocityUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 
 public class CommentOnWorkflowActionlet extends WorkFlowActionlet {
 
 
-	private static List<WorkflowActionletParameter> paramList = null; 
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 1L;
+    private static List<WorkflowActionletParameter> paramList = null;
+
+    @Override
+    public synchronized List<WorkflowActionletParameter> getParameters() {
+        if (paramList == null) {
+            paramList = new ArrayList<WorkflowActionletParameter>();
+            paramList.add(new WorkflowActionletParameter("comment", "Workflow Comment", null, true));
+        }
+        return paramList;
+    }
+
+    public String getName() {
+        return "Comment on Workflow";
+    }
+
+    public String getHowTo() {
+
+        return "This actionlet allows you to add a comment on the workflow task.";
```

Comment:
i18n too

Reply:
Should but we don't have this in any actionlet. We need to put in a task to do i18n them all @jdotcms